core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -245,18 +249,20 @@ object SparkSubmit {
     // from each other if only one is specified, or exit early if they are at odds.
     if (clusterManager == YARN) {
       if (args.master == "yarn-standalone") {
-        printWarning("\"yarn-standalone\" is deprecated. Use \"yarn-cluster\" instead.")
+        printWarning("\"yarn-standalone\" is deprecated. Use \"yarn\" " +
```

Comment:
So, if someone says "--master yarn-standalone", it seems like they'll see 3 different warnings in different places, right?
- one in L231 above
- one here
- one in the driver's output (SparkContext also logs something)

This particular one seems a little too redundant now, probably should be removed.